src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -27,6 +27,9 @@ var HTMLDOMPropertyConfig = {
   // name warnings.
   Properties: {
     allowFullScreen: HAS_BOOLEAN_VALUE,
+    // IE only true/false iFrame attribute
+    // https://msdn.microsoft.com/en-us/library/ms533072(v=vs.85).aspx
+    allowTransparency: HAS_BOOLEAN_VALUE,
```

Comment:
Is adding `HAS_BOOLEAN_VALUE` always safe? It changes semantics (falsy get removed). Are any of these actually `true` by default?

Should we keep using this flag or should we introduce a new one?

Reply:
Ah, darn it. Yes. We need a new flag.